Automattic/wp-calypso
# Update NPS survey random display frequency

This PR updates the frequency at which the NPS survey is shown. It was randomly shown in 1/5000 sessions (defined as a load of Calypso), and with this PR that changes to 1/2000 sessions.

![screen shot 2017-06-01 at 11 05 49 am](https://cloud.githubusercontent.com/assets/1017839/26672680/4fc64a6a-46ba-11e7-8cc3-2dbed4d1f605.png)


To test,

- sanity check the code change, then
- open your browser's console, and enter the line `npsSurvey();` to trigger the survey notice in development.
